### ISSUE TITLE
`chan gh-release <semver>` doesn't work with `GITHUB_TOKEN` defined

### ISSUE THREAD

**Issue by michaeldbianchi** [geut/chan#67]
Sorry for the noise 😢 , I'm just trying to get `chan` set up in GitHub Actions with a project I'm managing and I'm running into small issues.

## Current behavior
```sh
 λ unset GITHUB_TOKEN
 λ chan gh-release 0.1.3
[chan] [gh-release] › ℹ  info      Uploading GitHub release...

 λ export GITHUB_TOKEN=<REDACTED>
 λ chan gh-release 0.1.3
[chan] [gh-release] › ℹ  info      Uploading GitHub release...
[chan] [gh-release] › ✖  error     TypeError: Cannot read property 'createRelease' of undefined
    at createGithubRelease (file:///Users/mbianchi/workspace/dev/dotfiles/config/yarn/global/node_modules/@geut/chan/src/commands/gh-release.js:84:26)
    at Object.handler (file:///Users/mbianchi/workspace/dev/dotfiles/config/yarn/global/node_modules/@geut/chan/src/commands/gh-release.js:55:9)
```

The usage of `GITHUB_TOKEN` is causing an error of the [createRelease command](https://octokit.github.io/rest.js/v18#repos-create-release), though I can't figure out why. The usage in [gh-release](https://github.com/geut/chan/blob/main/packages/chan/src/commands/gh-release.js#L84-L91) looks correct, but maybe something isn't set properly in the `data` object.

**Comment by michaeldbianchi** [comment]
On second look at this, it looks like the call needs to be `github.rest.repos.createRelease`. I'm not sure if something has changed since this was originally written

**Comment by dpaez** [comment]
Closed with #68 

New release out: `v3.2.2`
